.github/workflows/.reusable-docker-build.yml

```diff
@@ -0,0 +1,84 @@
+# reusable workflow
+name: Build Docker Image
+
+on:
+  workflow_call:
+    inputs:
+      registry-url:
+        type: string
+        description: Github container registry base URL
+        required: false
+        default: ghcr.io
+      file:
+        type: string
+        description: Path to the Dockerfile
+        required: true
+      slug:
```

Comment:
```suggestion
      image-slug:
```

Reply:
I've also never heard the term `slug` used for an image. Why not use e.g. `image-name` here?

Reply:
Renamed to `image-name`.